podman/domain/images_build.py

```diff
@@ -204,8 +204,16 @@ def _render_params(kwargs) -> dict[str, list[Any]]:
         if "labels" in kwargs:
             params["labels"] = json.dumps(kwargs.get("labels"))
 
-        if params["dockerfile"] is None:
-            params["dockerfile"] = f".containerfile.{random.getrandbits(160):x}"
+        def default(value, def_value):
+            return def_value if value is None else value
+
+        params["outputformat"] = default(
+            params["outputformat"], "application/vnd.oci.image.manifest.v1+json"
+        )
+        params["layers"] = default(params["layers"], True)
+        params["dockerfile"] = default(
+            params["dockerfile"], f".containerfile.{random.getrandbits(160):x}"
+        )
```

Comment:
Once you get the parameters with `get` these lines are not needed. you need to also document the defaults in the lines above for the `build` function

Reply:
What about the dockerfile? Just document that if omitted it is generated randomly? I do not understand why it is like this or what it is for

Reply:
> What about the dockerfile? Just document that if omitted it is generated randomly? I do not understand why it is like this or what it is for

`dockerfile` docs are fine imho, you are changing the parameters of `layers` and `outputformat` so you should ensure that the defaults are documented

Reply:
This should be ok, I've kept the comments very short.